src/Graph12/Graph12.tsx

```diff
@@ -0,0 +1,53 @@
+import { ProgressCircle } from "../components/ProgressCircle";
+const data = [
+	{
+		label: "MAGNA",
+
+		count: 878,
+	},
+	{
+		label: "VELIT",
+
+		count: 618,
+	},
+	{
+		label: "DOLOR",
+
+		count: 521,
+	},
+	{
+		label: "CUPLA",
+
+		count: 221,
+	},
+];
+const maxData = 1000;
+
+const percentages = data.map((item) =>
+	Math.round((item.count / maxData) * 100)
+);
+
+export function Graph12() {
+	return (
+		<div className="mt-2">
+			<div className="flex items-center gap-3 w-[250px] ml-3">
```

Comment:
I would remove this ml-3, it doesn't have to be 1:1 as it is on designs

Reply:
done